applications/asset_tracker_v2/src/main.c

```diff
@@ -271,6 +275,50 @@ static bool event_handler(const struct event_header *eh)
 	return false;
 }
 
+static bool is_agps_processed(void)
+{
```

Comment:
It might be useful to put this in nrf_cloud_agps.c / .h instead.

Reply:
Agreed. But I think Ill keep it as is for now. Lets revisit this after the release.